### PR TITLE
Add SepiaSearch

### LINKED ISSUE #2217: Add SepiaSearch
Here's the search engine: https://sepiasearch.org/

Here's its source code: https://framagit.org/framasoft/peertube/search-index/

They also wrote an article about it: https://framablog.org/2020/09/22/sepia-search-our-search-engine-to-promote-peertube/

### PR DESCRIPTION
**Pull request by 0xhtml**
## What does this PR do?

This PR adds the SepiaSearch engine (https://sepiasearch.org).

## Why is this change important?

SepiaSearch is a PeerTube search engine that searches through multiple instances of PeerTube. It seems like a nice engine to add.

## How to test this PR locally?

Test by searching e.g. `!sep test`.

## Related issues

Closes #2217

### CODE REVIEW

File: searx/engines/sepiasearch.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+from dateutil import parser, relativedelta
+from urllib.parse import urlencode
+from datetime import datetime
+
+categories = ['videos']
+paging = True
+language_support = True
+time_range_support = True
+safesearch = True
+supported_languages = [
```
Comment (dalf) [inline searx/engines/sepiasearch.py:20]: Can you add in a comment the source of this list ?

Comment (0xhtml) [reply searx/engines/sepiasearch.py:20]: Which source? For `supported_languages`?

**Comment by dalf** [comment]
Sorry for the delay.

LGTM.

even if it is possible to add more information:
```diff
diff --git a/searx/engines/sepiasearch.py b/searx/engines/sepiasearch.py
index f69d8043..bb2f2289 100644
--- a/searx/engines/sepiasearch.py
+++ b/searx/engines/sepiasearch.py
@@ -38,6 +38,15 @@ time_range_table = {
 }
 
 
+embedded_url = '<iframe width="540" height="304" src="{url}" frameborder="0" allowfullscreen></iframe>'
+
+
+def minute_to_hm(minute):
+    if isinstance(minute, int):
+        return "%d:%02d"%(divmod(minute, 60))
+    return None
+
+
 def request(query, params):
     params['url'] = base_url + '?' + urlencode({
         'search': query,
@@ -70,13 +79,19 @@ def response(resp):
         content = result['description']
         thumbnail = result['thumbnailUrl']
         publishedDate = parser.parse(result['publishedAt'])
+        embedded = embedded_url.format(url=result.get('embedUrl'))
+        author = result.get('account', {}).get('displayName')
+        length = minute_to_hm(result.get('duration'))
         url = result['url']
 
         results.append({'url': url,
                         'title': title,
                         'content': content,
+                        'author': author,
+                        'length': length,
                         'template': 'videos.html',
                         'publishedDate': publishedDate,
+                        'embedded': embedded,
                         'thumbnail': thumbnail})
 
     return results
```

File: searx/engines/sepiasearch.py (hunk 1 above)
Comment (0xhtml) [reply searx/engines/sepiasearch.py:20]: The source for `supported_languages` is https://framagit.org/framasoft/peertube/search-index/-/blob/master/client/src/views/Search.vue#L618-641

**Comment by dalf** [comment]
Thank you !

File: searx/engines/sepiasearch.py (hunk 1 above)
Comment (dalf) [reply searx/engines/sepiasearch.py:20]: I added the URL in the commit message:  https://github.com/searx/searx/commit/53c8d945b489b1185d89e02bbc6b6ea6b60a91da

**Comment by TheEvilSkeleton** [comment]
Thank you so much @0xhtml

**Pull request merged**